demo/customer-sdk-demo.html

```diff
@@ -3,8 +3,8 @@
 <head>
   <title>Demo SDK use</title>
   <script src="../bower_components/socket.io-client/socket.io.js"></script>
-  <script src="bower_components/bringg-js-sdk/BringgSDK.js"></script>
-  <script src="bower_components/jquery/dist/jquery.min.js"></script>
+  <script src="../BringgSDK.js"></script>
```

Comment:
in the demo it should be ../bower_components/bringg-js-sdk/BringgSDK.js.
maybe we should devise a way for more convenient use internally (i.e with the above path for the sdk, and with the google keys). see you can conjure up something